src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlDatabase.md

```diff
@@ -285,24 +301,8 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -SampleName
-The name of the sample schema to apply when creating this database.
-
-```yaml
-Type: String
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
 ### CommonParameters
-
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
@cormacpayne correct, a previous PR added in the enclosed brackets to make the text a link.  I would like to keep it that way during future re-generation of the doc files.

What are your thoughts?